### PR TITLE
update(falco-metrics): falco metrics 0.39.0

### LINKED ISSUE #1368: Document Falco metrics update for Falco 0.39.0
/area documentation

**What would you like to be added**:

Reflect changes to the metrics and their naming in the website

**Why is this needed**:

### PR DESCRIPTION
**Pull request by incertum**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind bug

/kind cleanup

> /kind design

> /kind user-interface

> /kind content

> /kind event

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area blog

/area documentation

> /area community

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #1368

**Special notes for your reviewer**:

### CODE REVIEW

File: content/en/docs/metrics/falco-metrics.md [hunk 1]
```diff
@@ -28,6 +28,8 @@ metrics:
   resource_utilization_enabled: true
   state_counters_enabled: true
   kernel_event_counters_enabled: true
+  # Enabling `kernel_event_counters_per_cpu_enabled` automatically enables `kernel_event_counters_enabled`
+  kernel_event_counters_per_cpu_enabled: false
```
Comment (incertum) [inline content/en/docs/metrics/falco-metrics.md:32]: CC @Andreagit97

**Comment by incertum** [comment]
@leogr @FedeDP @LucaGuerra and @sgaist if changes are needed would it be possible for you to commit them? I won't be able to continue working on this PR. Thanks for backing me up!

@Issif would appreciate another close look regarding the Prometheus metrics, thanks a bunch in advance!

**Comment by LucaGuerra** [comment]
/hold

until release

**Review by LucaGuerra — APPROVED** [review]

**Pull request merged**